src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +56,243 @@ contract DssSpellCollateralAction {
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    uint256 constant ZERO_PCT_RATE           = 1000000000000000000000000000;
+    uint256 constant ZERO_ZERO_FIVE_PCT_RATE = 1000000000015850933588756013;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+    uint256 public constant RAY = 10**27;
+    uint256 public constant RAD = 10**45;
+
+    // -- RWA008 MIP21 components --
+    address constant RWA008                    = 0x30434AA15F85598F45406d08A79dCdD9a79335e9;
+    address constant MCD_JOIN_RWA008_A         = 0x4ce65E856f824C2b4a2eeD88E79b839eB366967d;
+    address constant RWA008_A_URN              = 0x6351915f840937Edba75656727f20165185FaB83;
+    address constant RWA008_A_URN_CLOSE_HELPER = 0xb220461eFFa9c0b1D09047D08Bf116DfbD9814fA;
+    address constant RWA008_A_INPUT_CONDUIT    = 0x6298a2498b4E3789bb8220Cf5f5b44f518509e2a;
+    address constant RWA008_A_OUTPUT_CONDUIT   = 0x1FE95E519F0CE8eCF1cdC885f4DeA7913e146149;
+    // SocGen's wallet
+    address constant RWA008_A_OPERATOR         = 0x3F761335890721752476d4F210A7ad9BEf66fb45;
+    // DIIS Group wallet
+    address constant RWA008_A_MATE             = 0xb9444802F0831A3EB9f90E24EFe5FfA20138d684;
+
+    uint256 constant RWA008_A_INITIAL_DC       = 30_000_000 * RAD;
+    uint256 constant RWA008_A_INITIAL_PRICE    = 30_437_069 * WAD;
+    uint48 constant  RWA008_A_TAU              = 0;
+
+    uint256 constant RWA008_REG_CLASS_RWA      = 3;
+
+    string constant RWA008_DOC                 = "QmdfzY6p5EpkYMN8wcomF2a1GsJbhkPiRQVRYSPfS4NZtB";
+    // -- RWA008 end --
+
+    // -- RWA009 MIP21 components --
+    address constant RWA009                  = 0xd2B101854F64Df738bA601840279838568583F39;
+    address constant MCD_JOIN_RWA009_A       = 0x7122B934F02A15954282Ed41572Ada539864773a;
+    address constant RWA009_A_URN            = 0xD2C8588C72026171Ec3a17369ad0f0734E30915d;
+    address constant RWA009_A_JAR            = 0xa484C16D2Ca15706c4B875710d9e80b7F101572B;
+    // Goerli: CES Goerli Multisig / Mainnet: Genesis
+    address constant RWA009_A_OUTPUT_CONDUIT = 0x7a3D23Dc73F7ead55399597aAE6e525b3DF95A88;
```

Comment:
Out of curiosity, do we have a signed message from the Genesis address in the forums.  I know this one is a CES multisig, but we will want strong confirmation of this address for mainnet.

Reply:
Genesis traders don't have access to the signing key to be able to sign arbitrary messages.

We asked them to send DAI to a specific address to prove they own it. I'm looking for the relevant Discord link that document how this happened.

Reply:
@godsflaw CES completed a test of sending 10DAI to the output conduit account and to demonstrate they had control of this account we asked them to send 10 DAI back . Etherscan transactions here:  https://etherscan.io/tx/0x4fd33ff92d222e6a843e18ce6c2180414dca30dab791f5b00830ee54f069ce81 and https://etherscan.io/tx/0xfecb71d9fa27e4b7b78fe31d0b7778a005ef96412d7d702371d334cab0092f1d  . We were unable to get them to sign a message from that account - because of operational restrictions on private keys which are managed by their custody provider.